docs/content/068-remote-commands.adoc

```diff
@@ -0,0 +1,37 @@
+[[remote-commands]]
+
+=== Remote Commands
+
+Operations to manage a remote store
+
+==== geowave remote calcstat
+
+include::manpages/geowave-calcstat.txt[]
+
+==== geowave remote clear
+
+include::manpages/geowave-clear.txt[]
+
+==== geowave remote listadapter
+
+include::manpages/geowave-listadapter.txt[]
+
+==== geowave remote listindex
+
+include::manpages/geowave-listindex.txt[]
+
+==== geowave remote liststats
+
+include::manpages/geowave-liststats.txt[]
+
+==== geowave remote recalcstats
+
+include::manpages/geowave-recalcstats.txt[]
+
+==== geowave remote rmadapter
```

Comment:
I'll leave the .txt file so it's there if we expose the command again. I'll remove the reference from the adoc so it doesn't show up in the pdf or html. Sound good? Alternatively I can just take out that .txt file as well.